test/run

```diff
@@ -5,4 +5,12 @@ set -eu
 # arguments but with an appropriate $TEST_OS, and optionally $TEST_SCENARIO
 
 [ -z "${TEST_SCENARIO:-}" ] || export TEST_BROWSER="$TEST_SCENARIO"
+
+# In case this is called with "prepare" as argument, only setup the test
+# environment without running the tests
+if [ x"${1:-}" = "xprepare" ]; then
```

Comment:
Can we please drop this? test/run is not an entry point for humans (just for CI), so it duplicates functionality which never gets exercised anywhere.